packages/babel-plugin-transform-react-jsx/src/index.js

```diff
@@ -1,100 +1,18 @@
+/* eslint-disable-next-line @babel/development/plugin-name */
```

Comment:
Is this needed? I thought that we would need it at line 6, or disable it for the whole file :thinking:

Reply:
The lint rule complains because `transform-classic` and `transform-automatic` are not Babel plugins. We could disable it for the whole file but it seems more specific to only disable it on the lines where we need it.